CRVResponse/standalone/wls-build/build.sh

```diff
@@ -1,5 +1,4 @@
-/grid/fermiapp/products/mu2e/artexternals/cmake/v2_8_12_2/Linux64bit+2.6-2.12/bin/cmake -DGeant4_DIR=$GEANT4_FQ_DIR/lib64/Geant4-10.2.3/ ../wls
-#/grid/fermiapp/products/mu2e/artexternals/cmake/v2_8_12_2/Linux64bit+2.6-2.12/bin/cmake -DGeant4_DIR=/cvmfs/mu2e.opensciencegrid.org/artexternals/geant4/v4_10_2_p03e/Linux64bit+2.6-2.12-e15-prof/lib64/Geant4-10.2.3/ ../wls
+/cvmfs/mu2e.opensciencegrid.org/artexternals/cmake/v3_17_3/Linux64bit+3.10-2.17/bin/cmake -DGeant4_DIR=/cvmfs/mu2e.opensciencegrid.org/artexternals/geant4/v4_10_6_p02b/Linux64bit+3.10-2.17-e19-mt-prof/lib64/Geant4-10.6.2/ ../wls
 
 # echo "CXX_DEFINES += " $1 >> CMakeFiles/wls.dir/flags.make
 
```

Comment:
How is the code in 'standalone' used?

Reply:
It is used for testing and tuning the scintillator parameters, and to create the Crv lookup tables. It runs independently of Offline and uses none of the Offline/art/mu2e infrastructure, but uses some code from the CRVResponse part of Offline (in order to avoid duplicated code).